### PR TITLE
Fix some formatting inconsistencies

### PR DESCRIPTION
**Pull request by AaronDewes**
I've tried automatically parsing the docs (for a WIP TypeScript client libarary which is automatically generated from docs + schemas), and these inconsistencies made it harder to do that.

### CODE REVIEW

**Review by vincenzopalazzo — APPROVED** [review]
ack https://github.com/ElementsProject/lightning/pull/5018/commits/c9f3fbae3fc202ffdd1544def2863d0203a8887f

**Comment by rustyrussell** [comment]
Actually, the \\ seem unnecessary, can we just get rid of all of them?

File: doc/lightning-connect.7.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ lightning-connect -- Command for connecting to another lightning node
 SYNOPSIS
 --------
 
-**connect** *id* \[*host* *port*\]
+**connect** *id* \[*host*\] \[*port*\]
```
Comment (rustyrussell) [inline doc/lightning-connect.7.md:7]: Actually, this should be '[host [port]]'

**Comment by cdecker** [comment]
nack https://github.com/ElementsProject/lightning/pull/5018/commits/c9f3fbae3fc202ffdd1544def2863d0203a8887f

**Comment by AaronDewes** [comment]
> Actually, the \ seem unnecessary, can we just get rid of all of them?

I think they make sense to prevent some markdown parsers from thinking this is a link, but I'll remove them, they can be readded if it causes any issues.

**Comment by rustyrussell** [comment]
> nack [c9f3fba](https://github.com/ElementsProject/lightning/commit/c9f3fbae3fc202ffdd1544def2863d0203a8887f)

Hmm, did you mistype, or did you have some feedback to share?

**Comment by rustyrussell** [comment]
Minor regeneration fix...

**Comment by cdecker** [comment]
Needs to be squashed, otherwise good to go 👍

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/ElementsProject/lightning/pull/5018/commits/96ddf9d426cd6823bf12f09f5a0f951f04dc378

**Comment by cdecker** [comment]
Rebased and squashed on top of current `master`

**Comment by rustyrussell** [comment]
Rebased again (should fix flakes!)

**Comment by rustyrussell** [comment]
Ack https://github.com/ElementsProject/lightning/commit/f4d93e4533699665bdd2945a9e9369d4f26129ba

**Pull request merged**